clusterloader2/pkg/flags/flags.go

```diff
@@ -65,6 +66,18 @@ func StringArrayVar(s *[]string, flagName string, defaultValue []string, descrip
 	pflag.StringArrayVar(s, flagName, defaultValue, description)
 }
 
+// StringSliceEnvVar creates a string slice flag with the given parameters.
```

Comment:
BTW - this support for string-slice flags, is orthogonal to the overal change, so if you could split that into separate PR, we could merge it now :)

Reply:
Was done some time ago :)